### PR TITLE
feature(4th-zone): Geography capacity extention to 4th zone

### PR DESCRIPTION
**Pull request by sreekarbvibm**

### CODE REVIEW

**Comment by sreekarbvibm** [comment]
`data "ibm_is_zones" "example" {
  region = "us-south"
}
output "zones_list" {
  value = data.ibm_is_zones.example
}

data "ibm_is_zone" "example" {
  name   = "us-south-1"
  region = "us-south"
}

output "zones_south1" {
  value = data.ibm_is_zone.example
}`

Screenshot 2024-06-19 at 1 22 30 PM
[terraform_4th_zone.log](https://github.com/user-attachments/files/15897436/terraform_4th_zone.log)

**Comment by sreekarbvibm** [comment]
[terraform_4th_zone.log](https://github.com/user-attachments/files/15897443/terraform_4th_zone.log)

File: ibm/service/vpc/data_source_ibm_is_zones.go [hunk 1]
```diff
@@ -35,6 +36,34 @@ func DataSourceIBMISZones() *schema.Resource {
...
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						isZoneName: {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						isZoneUniversalName: {
+							Type:     schema.TypeString,
+							Computed: true,
+							Optional: true,
```
Comment (deepaksibm) [inline ibm/service/vpc/data_source_ibm_is_zones.go:52]: Optional should be removed. Data sources have all Computed only fields

Comment (sreekarbvibm) [reply ibm/service/vpc/data_source_ibm_is_zones.go]: Done

File: ibm/service/vpc/data_source_ibm_is_zone.go [hunk 2]
```diff
@@ -14,6 +14,9 @@ const (
 	isZoneName   = "name"
```
Comment (ujjwal-ibm) [inline ibm/service/vpc/data_source_ibm_is_zone.go:14]: test files missing

**Review by astha-jain — APPROVED** [review]
Please add relevant tests and move to upstream

**Comment by ujjwal-ibm** [comment]
```
% tf state show data.ibm_is_zone.testacc_ds_zone
# data.ibm_is_zone.testacc_ds_zone:
data "ibm_is_zone" "testacc_ds_zone" {
    data_center = "DAL14"
    id          = "us-south.us-south-4"
    name        = "us-south-4"
    region      = "us-south"
    status      = "available"
}
% tf state show data.ibm_is_zones.testacc_ds_zones2
# data.ibm_is_zones.testacc_ds_zones2:
data "ibm_is_zones" "testacc_ds_zones2" {
    id        = "2024-09-17 02:57:31.438461 +0000 UTC"
    region    = "us-south"
    zone_info = [
        {
            data_center    = "DAL10"
            name           = "us-south-1"
            status         = "available"
            universal_name = ""
        },
        {
            data_center    = "DAL12"
            name           = "us-south-2"
            status         = "available"
            universal_name = ""
        },
        {
            data_center    = "DAL13"
            name           = "us-south-3"
            status         = "available"
            universal_name = ""
        },
        {
            data_center    = "DAL14"
            name           = "us-south-4"
            status         = "available"
            universal_name = ""
        },
    ]
    zones     = [
        "us-south-1",
        "us-south-2",
        "us-south-3",
        "us-south-4",
    ]
}
```

**Comment by sreekarbvibm** [comment]
Moving to upstream - https://github.com/IBM-Cloud/terraform-provider-ibm/pull/5644

**Pull request closed without merging**